cmd/code/scan.go

```diff
@@ -0,0 +1,83 @@
+package code
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/safedep/vet/internal/ui"
+	"github.com/safedep/vet/pkg/code"
+	"github.com/safedep/vet/pkg/common/logger"
+	"github.com/safedep/vet/pkg/storage"
+	"github.com/spf13/cobra"
+)
+
+var (
+	dbPath                    string
+	dirsToWalk                []string
+	importDirs                []string
+	skipDependencyUsagePlugin bool
+)
+
+func newScanCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "scan",
+		Short: "Scan source code",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			startScan()
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVar(&dbPath, "db", "vet.db", "Path to create the sqlite database")
```

Comment:
Lets keep this empty and mark as required param. Creating `vet.db` in current directory by default will lead to poor experience and mistakes i.e. ending up overwriting existing DBs

Reply:
makes sense, fixed it